webapp/components/file_preview.jsx

```diff
@@ -39,7 +39,7 @@ export default class FilePreview extends React.Component {
                 previewImage = (
                     <img
                         className='file-preview__image'
-                        src={getFileUrl(info.id)}
+                        src={getFilePreviewUrl(info.id)}
```

Comment:
Despite the name of the function, I think this should actually use `getFileThumbnailUrl` since that's what will be displayed on the post after it's updated